ocs_ci/ocs/resources/ocs.py

```diff
@@ -224,7 +228,18 @@ def get_ocs_csv():
         subscription_plan_approval=subscription_plan_approval,
     )
     channel = config.DEPLOYMENT.get("ocs_csv_channel")
-    ocs_csv_name = ocs_package_manifest.get_current_csv(channel=channel)
+    ocs_csv_name = None
+    # OCS CSV is extracted from the available CSVs in cluster namespace
+    # for Openshift dedicated platform
+    if config.ENV_DATA["platform"].lower() == "openshiftdedicated":
```

Comment:
please save `"openshiftdedicated"` in constants.py

Reply:
Done